tensorflow/inference/docker/2.2.0/py3/Dockerfile.gpu

```diff
@@ -15,7 +15,7 @@ ARG OPENSSL_VERSION=1.1.1g
 ARG TFS_SHORT_VERSION=2.2
 ARG TFS_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/serving/r2.2_aws/20200611-220145/gpu/py37/tensorflow_model_server
 
-ENV NCCL_VERSION=2.6.4-1+cuda10.2
+ENV NCCL_VERSION=2.7.6-1+cuda10.2
```

Comment:
Is this change to avoid shm issue for horovod ?

Reply:
It could have a chance to avoid the shm issue, but needs further test to verify it.